cardano-node/src/Cardano/CLI/Byron/Parsers.hs

```diff
@@ -181,6 +211,23 @@ parseSoftforkRuleParam = optional $
            <*> (rationalToLovelacePortion <$> parseFraction "softfork-thd-dec" "Propose threshold decrement (threshold will decrease by this amount after each epoch)")
         )
 
+parseSoftwareVersion :: Parser SoftwareVersion
+parseSoftwareVersion =
+  SoftwareVersion <$> parseApplicationName <*> parseNumSoftwareVersion
+
+parseApplicationName :: Parser ApplicationName
+parseApplicationName = ApplicationName <$> strOption
```

Comment:
`--application-name`  is not mentioned in `Available options` section of help for `Create a Byron era update proposal` cmd.

  Create a Byron era update proposal.

> Available options:
>   --config NODE-CONFIGURATION
>                            Configuration file for the cardano-node
>   --signing-key FILEPATH   Path to signing key.
>   --protocol-version-major WORD16
>                            Protocol verson major
>   --protocol-version-minor WORD16
>                            Protocol verson minor
>   --protocol-version-alt WORD8
>                            Protocol verson alt
>   --software-version-num WORD32
>                            Numeric software version associated with application
>                            name
>   --system-tag STRING      Identify which system (linux, win64, etc) the update
>                            proposal is for.
>